frontend/src/toolbar/experiments/experimentsTabLogic.tsx

```diff
@@ -305,6 +315,38 @@ export const experimentsTabLogic = kea<experimentsTabLogicType>([
                             })
                         }
                     })
+
+                    selectedVariant.transforms?.forEach((transform) => {
+                        if (transform.selector) {
+                            const undoTransform = {
+                                selector: transform.selector,
+                            } as unknown as WebExperimentTransform
```

Comment:
nit: it'd be nice to, down the line, make it so that when we modify experiments through transforms, we did it in such a way where we didn't have to throw out type safety with these assertions.  I know what the data structures look like and I don't have a problem with how you do this, but it's still a bit risky and there's probably a better way to model this data.

Reply:
Good point, I'll fill out the other properties and retain the type here.